types/node/index.d.ts

```diff
@@ -768,7 +768,14 @@ declare module "http" {
     import * as events from "events";
     import * as net from "net";
     import * as stream from "stream";
-    import { URL } from "url";
+
+    export interface OutgoingHeaders {
```

Comment:
personally I don't care that much as I would get hit just at one or two places. but others may see more impact.